core/foundation/inc/ROOT/RStringView.hxx

```diff
@@ -55,6 +55,23 @@ namespace std {
 
 #endif // ifdef else R__HAS_STD_STRING_VIEW
 
+#ifndef R__HAS_OP_EQUAL_PLUS_STRING_VIEW
+
+#include <string>
+
+namespace std {
+inline namespace __ROOT {
+
+inline std::string &operator+=(std::string &left, std::string_view&right)
```

Comment:
Where did you find this signature? Should `right` not be `const` or by-value?
```suggestion
inline std::string &operator+=(std::string &left, const std::string_view &right)
```

Reply:
being it `string_view`, I think it should be by value

Reply:
Marco is right 'https://en.cppreference.com/w/cpp/string/basic_string_view/operator_cmp' and I flip-floped between the 2 and ended up pushing the wrong one :( .. fixing.